mobile/src/components/setting/common/constants.ts

```diff
@@ -3,6 +3,7 @@ import { UserDetailsUpdateInfoRequestDto } from '~/common/types/types';
 
 const SELECTION_LIMIT = 1;
 const AVATAR_MAX_SIZE = 1000000; // 1MB
+const FILE_FORMATS = ['image/png', 'image/jpeg', 'image/svg+xml', 'image/svg'];
```

Comment:
Don't we have such formats in ContentType?

Reply:
Oh, thanks)